eyieko/landville-frontend
# #168072255 Display clients list

## Description ##
Displays a list of clients who sell property on the platform
## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## How can this PR be manually tested: ##
- Checkout to brach `ft-display-clients-list-168072255`
- Run the local server with the command `ng serve -o`
- After you have logged in successfully, you should see `Partners` link in the navigation 
- This should redirect you to the clients' page that displays client companies. Otherwise, a toast message should inform you in case of any errors encountered.

## Any background context you want to provide? ##
A Partners link should be on the navbar, leading to the list of client. On the Client list, the available information should include the client name, a link to view client reviews and a link to add a review for the client. If an unauthenticated user tries to add a review, they should be redirected to the login page. Unauthenticated users should be able to view reviews about any client.

## Related Pivotal Tracker stories ##
[#168072255](https://www.pivotaltracker.com/story/show/168072255)
